nilearn/decomposition/tests/conftest.py

```diff
@@ -115,9 +115,9 @@ def _decomposition_images_surface(
 def _decomposition_img(
     data_type,
     rng,
-    mesh,
-    shape,
-    affine,
+    mesh=None,
```

Comment:
Why do these arguments default to `None` ?

Reply:
Mostly added those to be able to use it, as a function outside of the pytests fixture context.

When use in tests with fixtures mesh, shape and affine are always defined even if we only need to generate some surface or some volume data.

But here I actually needed to only generate some surface data, so it felt strange to also pass some affine and shape just to 'comply' with the function signature.

Setting those parameters to None makes this a bit easier.